docs/design/features/dotnet-pgo.md

```diff
@@ -37,14 +29,14 @@ set COMPLUS_TC_QuickJitForLoops=1
 set COMPLUS_TC_CallCountThreshold=10000
 set COMPLUS_ReadyToRun=0
 
-dotnet-trace collect --providers Microsoft-Windows-DotNETRuntime:0x1E000080018:4 -- bin\Release\net6.0\pgotest.exe
+dotnet-trace collect --providers Microsoft-Windows-DotNETRuntime:0x1E000080018:4 -- bin\Release\net{version-number-goes-here}.0\pgotest.exe
```

Comment:
does this {version-number-goes-here} get substituted later on?

Reply:
I think it's inconsequential as it is just a markdown file. Introduced here: https://github.com/dotnet/runtime/pull/103927/files